helix-core/src/test/java/org/apache/helix/controller/rebalancer/waged/TestWagedRebalancer.java

```diff
@@ -112,7 +112,8 @@ protected ResourceControllerDataProvider setupClusterDataCache() throws IOExcept
   @Test
   public void testRebalance() throws IOException, HelixRebalanceException {
     _metadataStore.clearMetadataStore();
-    WagedRebalancer rebalancer = new WagedRebalancer(_metadataStore, _algorithm);
+    WagedRebalancer rebalancer =
+        new WagedRebalancer(_metadataStore, _algorithm, new DelayedAutoRebalancer(), null);
```

Comment:
Create a constructor without the last field?

Reply:
Sure.